app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/Tooltip.kt

```diff
@@ -72,8 +72,8 @@ fun HedvigTooltip(
     if (!showTooltip) return@LaunchedEffect
     delay(0.5.seconds)
     transientShowTooltip = showTooltip
-    tooltipShown()
     delay(5.seconds)
+    tooltipShown()
```

Comment:
So this was the way it was on purpose. It considered the tooltip as "shown" the moment it was in fact shown.

With this new change we are going to have this situation:
1. Member sees the tooltip as we show it.
2. 3 seconds later, they navigate away from this screen.
3. Then when they come back to this screen 
4. We show the tooltip again, since we never marked it as "seen"

That's why we marked it as seen asap, but then kept a "transient" state of if we show it or not, so that despite the fact that we have it marked as "seen" in the datastore or wherever we store this information